sphinx/writers/text.py

```diff
@@ -156,7 +156,7 @@ def cell_width(self, cell: Cell, source: list[int]) -> int:
         This takes into account cells spanning multiple columns.
         """
         if cell.row is None or cell.col is None:
-            msg = 'Cell co-ordinates have not been set'
+            msg = 'Cell coordinates have not been set'
```

Comment:
`co-ordinates` is fine

Reply:
Both `co-ordinate` and `coordinate` are fine, especially in British English according to dictionaries. However, the [Google Ngram Viewer](https://books.google.com/ngrams/graph?content=co-ordinate%2Ccoordinate&year_start=1800&year_end=2019&corpus=en-2019) shows `coordinate`  in wider use, including in the [British English corpus](https://books.google.com/ngrams/graph?content=co-ordinate%2Ccoordinate&year_start=1800&year_end=2019&corpus=en-GB-2019) strangely enough.

It's up to you, but what's wrong with following common use when targetting a large and international audience?

Reply:
Even though co-ordinate is fine, I personally prefer one without "-". Also it removes one character from the code base so it's technically an optimization!

Reply:
How about a compromise and use "coördinates" like the New Yorker! 🙃

https://www.newyorker.com/culture/culture-desk/the-curse-of-the-diaeresis